src/3_branch.js

```diff
@@ -106,7 +107,7 @@ Branch = function() {
 		}
 		return default_branch;
 	}
-	this._queue = Queue();
+	this._queue = new Queue();
```

Comment:
It's caused by bad naming, this `Queue` is not a constructor, but when named upper-camelly, jshint thinks it is.

Renamed to task_queue (queue doesn't work because it introduces another global-local variable name conflict)